README.md

```diff
@@ -12,6 +12,7 @@
     <a href="http://badge.fury.io/js/jest"><img src="https://badge.fury.io/js/jest.svg" alt="npm version"></a>
     <a href="https://dependabot.com/compatibility-score.html?dependency-name=jest&amp;package-manager=npm_and_yarn&amp;version-scheme=semver"><img src="https://api.dependabot.com/badges/compatibility_score?dependency-name=jest&amp;package-manager=npm_and_yarn&amp;version-scheme=semver" alt="SemVer">
     <a href="https://twitter.com/acdlite/status/974390255393505280"><img src="https://img.shields.io/badge/speed-blazing%20%F0%9F%94%A5-brightgreen.svg" alt="Blazing Fast"></a>
+    <a href="https://github.com/facebook/jest/pulls"><img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square" alt="PR's welcome"></a>
```

Comment:
can we use a rounded version of the badge to fit the others?
<img width="258" alt="screenshot 2018-11-08 at 12 06 28" src="https://user-images.githubusercontent.com/5106466/48194986-c9f48680-e34e-11e8-9ddb-30d8c88bbe46.png">

Reply:
Fixed 👍